docs/users-guide/events-and-timelines.md

```diff
@@ -0,0 +1,62 @@
+# Events and timelines
+
+A lot of discussions around data have a moment when someone asks a question related to a specific point in time: "Wait, what's the spike in March again?", or "When did the new widget launch?"
+
+Events and timelines are a way to capture that chronological knowledge and make it available when you need it, in context (that is, when you're viewing a chart). Events are a great way to store institutional knowledge about what happened and when, so people (including yourself three months from now) won't have to figure out (again) why the line chart spiked back in March.
+
+## Events
+
+Events are basically dates + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
+
+## Timelines
+
+You can group events into timelines, and associate those timelines with [collections](collections.md). For example, you may want to have a timeline that contains important email or sales dates, or an outages timeline that tracks downtime. You can move events between timelines, and move timelines from collection to collection.
+
+Collections can have timelines, and timelines can contain events. When we say, "collections have timelines", in practice what this means is that when you view a time series question in that collection, you'll see events from all the collection's timelines plotted on its chart. Events are invisible on dashboards.
```

Comment:
Something about the structure of this paragraph is confusing to me - maybe the "time series question in _that_ collection" <-- what is "that" referring to? Tried a couple different ways to rewrite it but nothing was sticking. Can it be rephrased or switched around somehow? Is this close to it?:

"Collections can have timelines, and timelines can contain events. In practice what this means is that events you've added to a timeline will show up on time series questions stored in the same collection as that timeline. Events are invisible on dashboards."